diamm/templates/website/archive/archive_list.jinja2

```diff
@@ -1,7 +1,7 @@
 {% extends "base.jinja2" %}
 
 {% block body %}
-    <div class="col-md-12">
+    <div>
```

Comment:
You should look at this again, and consult the [Skeleton CSS docs](http://getskeleton.com) for the CSS grid system. Specifically, you will need the 'class="row"' bits, and nested divs with 'class="twelve columns" or "sixteen columns".